dist/tools/uf2/Makefile

```
@@ -0,0 +1,10 @@
+PKG_NAME=UF2
+PKG_URL=https://github.com/microsoft/uf2.git
+PKG_VERSION=41394bd604e341ebcfdf0a8c4acce9ec8fd73d93
+PKG_LICENSE=MIT
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all:
```

```suggestion
all: $(CURDIR)/uf2conv.py
```
(untested)
The idea is to not copy (and make it executable) again the script if it's already there.

---

That won't work:

```
make[1]: *** No rule to make target '/home/benpicco/dev/RIOT/dist/tools/uf2/uf2conv.py', needed by 'all'.  Stop.
```

---

Ah yes, I forgot something :)

```mk
all: $(CURDIR)/uf2conv.py

$(CURDIR)/uf2conv.py:
	cp $(PKG_SOURCE_DIR)/utils/uf2conv.py .
	chmod a+x uf2conv.py
```

---

ah yes that works!